python/py-pyaudio/Portfile

```diff
@@ -46,12 +47,13 @@ if {${name} ne ${subport}} {
     python.link_binaries no
 
     post-destroot {
-        xinstall -d ${destroot}${prefix}/share/doc/py${python.version}-pyaudio
-        xinstall -m 640 {*}[glob ${worksrcpath}/examples/*] \
+        xinstall -d ${destroot}${prefix}/share/doc/py${python.version}-pyaudio/examples
+        xinstall -m 644 ${worksrcpath}/README \
             ${destroot}${prefix}/share/doc/py${python.version}-pyaudio
-        xinstall -d ${destroot}${prefix}/share/examples/py-pyaudio
-        xinstall -m 640 {*}[glob ${worksrcpath}/examples/*] \
-            ${destroot}${prefix}/share/examples/py-pyaudio
+        xinstall -m 644 ${worksrcpath}/CHANGELOG \
+            ${destroot}${prefix}/share/doc/py${python.version}-pyaudio
+        xinstall -m 644 {*}[glob ${worksrcpath}/examples/*] \
+            ${destroot}${prefix}/share/doc/py${python.version}-pyaudio/examples
```

Comment:
you don't need two lines to install README and CHANGELOG if you use the ```-W``` flag (see documentation [here](https://trac.macports.org/wiki/PortfileRecipes#doc)). So this could become instead:
```
    post-destroot {
        set docdir ${prefix}/share/doc/${subport}
        xinstall -d ${destroot}${docdir}
        xinstall -m 0644 -W ${worksrcpath} README CHANGELOG \
            ${destroot}${docdir}
        xinstall -m 0644 {*}[glob ${worksrcpath}/examples/*] \
            ${destroot}${docdir}/examples
    }
```